### ISSUE TITLE
Option to revert to the previous file staging menu

### ISSUE THREAD

**Issue by Minabsapi** [Murmele/Gittyup#90]
This new Double Tree View of the file staging menu may look a little cleaner to some than the one showcased in GitAhead, but I wouldn't really call it a complete upgrade. Some features available in the previous version have disappeared (stage/unstage multiple files at once by ctrl/shift - clicking then right click to bring the menu up, discard changes in this same menu...), and the new separation between staged and unstaged files makes it harder to simply toggle one staged then unstaged (which is sometimes necessary when you staged a file too soon)

The times this gets a little polishing, could we please get the previous file staging menu, at least as an option ?

**Comment by Murmele** [comment]
I agree with you, it is not that consequently integrated into the gui. The benefit of this view is you see which documents are staged and unstaged and you don't forget one. 
I can try if it is easy to add the other view again.

**Comment by Murmele** [comment]
The commit is: 7b13d6a4a61ab499067b9ef31416bdaffbfadace
Currently I don't have the time to revert it

**Comment by stefanknotzer** [comment]
What about a simple checkbox or context menu button to switch between double tree view (default) to a single tree view (like when viewing committed files)?

This should be easy to implement and does not bring back dead code...
Also the "disappeared" features can be brought back (file context menu, selection model)

**Comment by Murmele** [comment]
Or should we implement it into the current DoubleTreeWidget? Don't know what is less work

**Comment by stefanknotzer** [comment]
I like the idea and am implementing a context menu button in DoubleTreeWidget:
- switch between single and double tree view
- hide folders (= the result should be a file list view) and allow multiple selections
Then you can choose between 1 or 2 file trees or file lists.

Or maybe even change the tree and list view separately for the 2 views - I could add a little button next to the Collapse/Expand button with a tree or list symbol to toggle the view style.
